packages/itwinui-react/src/core/InformationPanel/InformationPanel.tsx

```diff
@@ -60,8 +60,13 @@ export const InformationPanel = React.forwardRef((props, ref) => {
     ...rest
   } = props;
 
+  const [infoPanelSize, setInfoPanelSize] = React.useState<{
+    width: number | undefined;
+    height: number | undefined;
+  }>({ width: undefined, height: undefined });
```

Comment:
I just realized that we don't even need to store both dimensions because we are only using one at a time, based on the ternary styles

```js
  const [infoPanelSize, setInfoPanelSize] = React.useState<number>(undefined);
```

Reply:
Okay, I'll update the state to be this instead.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1964/commits/499b86a1893cdd28e63d08803276cdfc182e5f84) I want to note that this change causes the size to be maintained after orientation change, it just changes how the value is used. This behavior should be fine, since if a user reduces the size of a panel and then changes its location, they probably don't expect the size to reset.

Reply:
can you also add the same explanation to a code comment in the test where you are testing this behavior?

Reply:
Will do.

Reply:
Added a comment in this [commit.](https://github.com/iTwin/iTwinUI/pull/1964/commits/ebc06196b0575adf72724056bcc9171ee8e18291)